src/main/java/domain/Lottery.java

```diff
@@ -0,0 +1,38 @@
+package domain;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import utils.LotteryMessage;
+
+public class Lottery {
+
+	private static final int LOTTERY_SIZE = 6;
+	private final List<LotteryNumber> numbers;
+
+	public Lottery(final List<LotteryNumber> numbers) {
+		validateSize(numbers.size());
+		this.numbers = numbers;
+	}
+
+
+	private void validateSize(final int size) {
+		if (size != LOTTERY_SIZE) {
+			throw new IllegalArgumentException(LotteryMessage.LOTTERY_SIZE_ERROR);
+		}
+	}
+
+	public int countSameNumber(final Lottery lottery) {
+		final List<LotteryNumber> differences = new ArrayList<>(this.numbers);
+		differences.removeAll(lottery.numbers);
+		return LOTTERY_SIZE - differences.size();
```

Comment:
```suggestion
                       (int) numbers.stream()
				.filter(lottery::contains)
				.count();
```
도 가능할거 같은데 위와 같이 한 이유가 있을까요??

Reply:
저 부분을 설계하면서 스트림을 사용할 생각을 하지 못했네요. 수정하겠습니다.

Reply:
1차 수정 완료